lang/mr.json

```diff
@@ -0,0 +1,98 @@
+{
+    "Audio Player": "ध्वनि वादक",
+    "Video Player": "चित्रवाणी वादक",
+    "Play": "वाजवा",
+    "Pause": "थांबा",
+    "Replay": "पुन्हा वाजवा",
+    "Current Time": "वर्तमान वेळ",
+    "Duration": "कालावधी",
+    "Remaining Time": "उर्वरित वेळ",
+    "Stream Type": "प्रवाह प्रकार",
+    "LIVE": "थेट प्रसारण",
+    "Seek to live, currently behind live": "थेट प्रसारणकडे जा, चालू क्षणाला थेट प्रसारण पासुन मागे आहे",
+    "Seek to live, currently playing live": "थेट प्रसारणकडे जा, चालू क्षणाला थेट प्रसारण आहे",
+    "Loaded": "सज्ज",
+    "Progress": "प्रगती",
+    "Progress Bar": "प्रगती पट्टा",
+    "progress bar timing: currentTime={1} duration={2}": "प्रगती पट्टा वेळ: {1} of {2}",
+    "Fullscreen": "संपूर्ण पडदा",
+    "Exit Fullscreen": "संपूर्ण पडद्यातून बाहेर पडा",
+    "Mute": "मुक",
+    "Unmute": "अमूक",
+    "Playback Rate": "Playback Rate",
```

Comment:
If there's not a translation it's better to just leave this out so that the [translations needed doc](https://github.com/videojs/video.js/blob/main/docs/translations-needed.md) will show that an update is needed. The default English string will be shown anyway in this case.